### ISSUE TITLE
Missing of softmax layer in the training of content encoder module

### ISSUE THREAD

**Issue by RamakrishnaChaitanya** [yuval-reshef/StreamVC#7]
Hi, 

It was mentioned in the paper that we need to apply Layer Normalization, Linear and Softmax on the outputs of content encoder. However, in your implementation (encoder_classifier.py), the softmax layer seems to be missing. Please let me know if i'm missing something. 

![image](https://github.com/user-attachments/assets/95f704bf-faaf-47d9-9de8-f4355dfa4065)
